### PR TITLE
Fix Nacos console Service list .Group search is not correct #4306

### LINKED ISSUE #4306: Nacos console Service list .Group search is not correct
Nacos console Service list .Group search is not correct.
WeChatf4c28205f9159b7be6b47a9123b47cff
I searched for 'Gateway' in 'group name' . There is no 'group name' with the name 'Gateway' in my data. But he was able to query the data.

### PR DESCRIPTION
**Pull request by woxiangwoxiangxiang**
Fix #4306

### CODE REVIEW

**Comment by paderlol** [comment]
Please append some unit test cases to prove your changes

**Comment by Maijh97** [comment]
@woxiangwoxiangxiang Please sign the Contributor License Agreement first.

**Comment by woxiangwoxiangxiang** [comment]
> Please append some unit test cases to prove your changes

Where do I add unit tests? In ServiceManagerTest?

But ServiceManagerTest doesn't run correctly.
WeChat459ff5a1fea75c6664dd9c5adb9eb531

**Comment by paderlol** [comment]
You need to verify the split is correct through unit tests

File: naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java [hunk 1]
```diff
@@ -923,7 +923,7 @@ public int getInstanceCount() {
         }
         return total;
     }
-    
+
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java:926]: Please do not change the indentation. Maybe you do not use nacos code style. Can you read the style/codeStyle.md in source code directory?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Comment by KomachiSion** [comment]
> > Please append some unit test cases to prove your changes
> 
> Where do I add unit tests? In ServiceManagerTest?
> 
> But ServiceManagerTest doesn't run correctly.
> WeChat459ff5a1fea75c6664dd9c5adb9eb531
> This is the result of the original ServiceManagerTest run

Why you need spy to create ServiceManager?

**Comment by KomachiSion** [comment]
@woxiangwoxiangxiang Please fix code style problem and sign CLA, otherwise we will close this PR.

**Comment by KomachiSion** [comment]
@woxiangwoxiangxiang Sorry, your CLA still not right, I see the true commit user is `wangchunhao` not your github id.
So you should use `git config user.name` and `git config user.email` to re-commit your change and re-submit a PR to fix it.

BTW, we strongly recommend you use the nacos code style when you change nacos codes.

**Pull request closed without merging**